src/content/docs/en/reference/errors/env-invalid-variable.mdx

```diff
@@ -13,6 +13,11 @@ import DontEditWarning from '~/components/DontEditWarning.astro'
 <DontEditWarning />
 
 
+:::caution[Deprecated]
+This error cannot be emitted by Astro anymore
```

Comment:
I think this wording is maybe a tiny bit confusing, because some people landing here will literally have just had this error emitted by Astro. So it is possible, if you are on an old enough version.

Maybe something like this would work better? (And in the other places it is reused.)

```suggestion
This error was emitted by an older version of Astro and is no longer in use.
```

Reply:
So not only *are* they getting that error, but we also know they are running an older version of Astro. Maybe some kind of message like below is additionally helpful?

> This error is from an older version of Astro and is no longer in use. If you are unable to upgrade your project to a more recent version, then you can consult [unmaintained snapshots of older documentation][(/en/upgrade-astro/#older-docs-unmaintained) for assistance.